From 3cb6bf770d8b0fda746e43300fd706ade4f24a50 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
Date: Wed, 20 Dec 2017 00:27:02 +0700
Subject: [PATCH 454/504] phy: rcar-gen2: fix warning message

Remove unused variables and correct return value in some functions.

Signed-off-by: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 drivers/phy/phy-rcar-gen2.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/drivers/phy/phy-rcar-gen2.c b/drivers/phy/phy-rcar-gen2.c
index 3961477..43954dd 100644
--- a/drivers/phy/phy-rcar-gen2.c
+++ b/drivers/phy/phy-rcar-gen2.c
@@ -111,7 +111,6 @@ static void rcar_gen2_phy_switch(struct rcar_gen2_channel *channel,
 	ugctrl2 |= select_value;
 	writel(ugctrl2, drv->base + USBHS_UGCTRL2);
 	spin_unlock_irqrestore(&drv->lock, flags);
-	return 0;
 }
 
 static int rcar_gen2_phy_init(struct phy *p)
@@ -149,6 +148,7 @@ static int rcar_gen2_phy_init(struct phy *p)
 	clk_prepare_enable(drv->clk);
 	rcar_gen2_phy_switch(channel, phy->select_value);
 #endif
+	return 0;
 }
 
 static int rcar_gen2_phy_exit(struct phy *p)
@@ -326,8 +326,7 @@ static void gpio_id_work(struct work_struct *work)
 {
 	struct rcar_gen2_channel *channel = container_of(work,
 					struct rcar_gen2_channel, work_id.work);
-	struct usb_phy *usbphy = &channel->usbphy;
-	int status, vbus, id;
+	int id;
 
 	id = !!gpio_get_value(channel->gpio_id);
 	/* Switch the PHY over */
@@ -422,7 +421,6 @@ static int probe_otg(struct platform_device *pdev,
 {
 	struct device *dev = &pdev->dev;
 	struct rcar_gen2_channel *ch = drv->channels;
-	int irq;
 	int ret;
 	/* GPIOs for Host/Fn switching */
 	ch->gpio_id = of_get_named_gpio_flags(dev->of_node,
-- 
2.7.4

